contracts/Reactions/ReactionVault.sol

```diff
@@ -799,4 +813,61 @@ contract ReactionVault is
         require(addressManager.roleManager().isAdmin(msg.sender), "Not Admin");
         payable(msg.sender).transfer(address(this).balance);
     }
+
+    /// @dev Allow "spending" a reaction without any value
```

Comment:
We may want to put a comment here letting the reader know that "spending a free reaction" is just triggering an event being recorded to the blockchain.  I was looking for more logic and got confused... ha.

Reply:
done, also renamed function to a better name: "_freeReaction()"